src/network/networkd-network.c

```diff
@@ -150,10 +169,21 @@ static int network_verify(Network *network) {
         if (network->ip_masquerade)
                 network->ip_forward |= ADDRESS_FAMILY_IPV4;
 
-        if (network->mtu > 0 && network->dhcp_use_mtu) {
-                log_warning("%s: MTUBytes= in [Link] section and UseMTU= in [DHCP] section are set. "
-                            "Disabling UseMTU=.", network->filename);
-                network->dhcp_use_mtu = false;
+        network_set_stacked_netdevs_mtu(network);
```

Comment:
`stacked_netdevs_mtu` is only used internally in this function. I think it's much nicer to to make it a "global" state in the Network structure, but simply have `network_set_stacked_netdevs_mtu` return it.